smf/sled-agent/non-gimlet/config-rss.toml

```diff
@@ -22,25 +23,82 @@ ntp_servers = [ "ntp.eng.oxide.computer" ]
 dns_servers = [ "1.1.1.1", "9.9.9.9" ]
 
 # Delegated external DNS zone name
+#
+# The rack provides separate external API and console endpoints for each Silo.
+# These are named `$silo_name.sys.$external_dns_zone_name`.  For a Silo called
+# "eng" with delegated domain "oxide.example", the API would be accessible at
+# "eng.sys.oxide.example".  The rack runs external DNS servers that serve A/AAAA
+# records for these DNS names.
+#
+# In a real deployment, the operator would own this DNS domain and delegate it
+# to the rack.  That means adding glue records in their DNS servers for the
+# parent domain that point to the rack's external DNS servers.
+#
+# In development, we usually use a fake domain here and configure clients to use
+# the rack's external DNS server as their normal DNS server.  Programmatically,
+# we do this with a custom reqwest Resolver.  Using curl or the Oxide CLI, you
+# can do this with the `--resolve` option.  In practice, people often skip DNS
+# altogether (which also means skipping TLS).
 external_dns_zone_name = "oxide.test"
 
+# The IP ranges configured as part of the "internal services" IP Pool
+#
+# This is a range of *external* IP addresses that will be assigned to the
+# External DNS server(s) and Nexus instances.  These are the IPs that users
+# *outside* the rack will use to reach these services.  These will also be used
+# by services like NTP as their externally-facing address for NAT.
+#
+# For more on this and what to put here, see docs/how-to-run.adoc.
+[[internal_services_ip_pool_ranges]]
+first = "192.168.1.20"
+last = "192.168.1.29"
+
+# TODO - this configuration is subject to change going forward. Ultimately these
+# parameters should be provided to the control plane via wicket, but we need to
+# put these parameters somewhere in the meantime so that the Nexus API will be
+# reachable upon Rack startup.
+
+# Configuration to bring up Boundary Services and make Nexus reachable from the
+# outside.  See docs/how-to-run.adoc for more on what to put here.
+[rack_network_config]
+# The gateway IP for the rack's external network
+gateway_ip = "192.168.1.199"
+# A range of IP addresses used by Boundary Services on the external network.  In
+# a real system, these would be addresses of the uplink ports on the Sidecar.
+# With softnpu, only one address is used.
+infra_ip_first = "192.168.1.30"
+infra_ip_last = "192.168.1.30"
+# Name of the uplink port.  This should always be "qsfp0" when using softnpu.
+uplink_port = "qsfp0"
+uplink_port_speed = "40G"
+uplink_port_fec="none"
+# For softnpu, an address within the "infra" block above that will be used for
+# the softnpu uplink port.  You can just pick the first address in that pool.
+uplink_ip = "192.168.1.30"
```

Comment:
@rcgoodfellow, @luqmana could you take a look at the comments I've added here?  Other changes here include moving some sections around so that the stuff people are likely to need to change are earlier in the file and changing the ranges in the internal service pool, infra pool, and the uplink_ip based on various suggestions in chat today.

Reply:
This looks good to me.